### DISCUSSION TITLE
Iris 1.1.3 for Minecraft 1.16.5, 1.17.1, and 1.18

### DISCUSSION THREAD

**Discussion by IMS212** [IrisShaders/Iris#1037] [General]
Iris version 1.1.3 for Minecraft 1.16.5, Minecraft 1.17.1, and Minecraft 1.18 has been released!

This release was jointly developed by coderbot, IMS, and Justsnoopy30, and also contains major contributions from Kroppeb and maximum#8760. It also contains many other smaller (but still important) contributions from many other people!

⬇ Download the release here: https://irisshaders.net/download.html

Here's what's changed since 1.1.2:

 Sodium is no longer bundled, and must be installed separately (The installer still installs it for you)
🌄 Iris now fully supports SEUS Renewed, and now has functional support (on some hardware / drivers) for SEUS PTGI & SEUS PTGI HRR
🖐 Hands now have proper shadows / lighting with most shader packs
🧙‍♀️Witches and drowned now have proper lighting with Complementary Shaders
🌌 Night sky effects now render properly with Complementary Shaders (galaxies if manually enabled, auroras) and AstraLex (planets)
💡 If manually enabled, Scene-Aware Colored Lighting now works properly on Complementary Shaders, even on macOS!
🌈 Rainbows and rain puddles now appear when using Complementary Shaders
🗣 A number of translations have been updated and tweaked
🩹 Patched a number of other minor bugs, crashes, and issues
Known potential issues:

⚠ Sodium 0.4.0-alpha5 for 1.18 is still in early alpha, meaning that you might run into significant bugs or other issues. Please exercise good issue-reporting skills if you want to report issues with the 1.18 versions of Iris / Sodium.
⬆ Remember to update to Immersive Portals 0.33 if you are using Immersive Portals on 1.17.1
⬆ If you are using Indium on 1.17.1+, you must update it to version 1.0.1 or later. Indium 1.0.0 is not compatible with Sodium 0.3.3.
A more detailed changelog is available for those interested: https://github.com/IrisShaders/Iris/blob/trunk/docs/changelogs/1.1.3/full.md

<hr /><em>This discussion was created from the release <a href='https://github.com/IrisShaders/Iris/releases/tag/1.16.x/v1.1.3'>Iris 1.1.3 for Minecraft 1.16.5, 1.17.1, and 1.18</a>.</em>

**Comment by KirbyMessiah** [comment]
Enabling galaxies option manually does not work for me on Complementary Shaders, but the auroras setting works. Has anyone able to get it to work? I am running iris 1.1.3 on minecraft 1.18.1.

**Reply by Xyberspace** [reply]
same issue and setup here
